tox.ini

```diff
@@ -8,6 +8,7 @@ requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
     tox-pypi-filter >= 0.12
+    virtualenv == 20.0.33
```

Comment:
Nope, can't do this either. `could not satisfy requires MissingDependency(<Requirement('virtualenv==20.0.33')>)`